index.html

```diff
@@ -1071,10 +1071,19 @@ <h2>RDF Serialization/Deserialization</h2>
       using information provided by the <a>active context</a>. This
       section describes how to produce an <a>active context</a>.</p>
 
+    <div id="change_api_638" class="candidate correction">
+      <span class="marker">Candidate Correction 8</span>
+      <p>Change the type of <a>term definition</a> from <a>array</a> to <a>map</a> in <a>active context</a>,
+        to be consistent with the way it is used in the algorithms.
+        Clarify the notion <a>term definition</a>.
+        For more information, refer to <a href="https://github.com/w3c/json-ld-api/issues/630">issue 630</a>.
+      </p>
+    </div>
+
     <p>The <a>active context</a> consists of:</p>
     <ul>
       <li>the active <a>term definitions</a> which specify how
-        keys and values have to be interpreted (<a>array</a> of <a>term definitions</a>),</li>
+        keys and values have to be interpreted (<del cite="#change_api_638"><a>array</a></del><ins cite="#change_api_638"><a>map</a></ins> of <a>term definitions</a>),</li>
```

Comment:
We probably need to say what the map key is for each entry (the term itself, of course).